app/views/invoices/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1> Invoice <%= @invoice.id %> </h1>
+
+<div class="invoice">
+  Invoice ID: <%= @invoice.id %> <br>
+  Invoice Status: <%= @invoice.status %> <br>
+  Invoice Created At: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %> <br>
+  Invoice Customer: <%= @invoice.customer.full_name %>
+</div>
+
+<br>
+
+<div class="items">
+  <% @invoice.items.each do |item| %>
+    <div id="item-<%=item.id%>">
+      Item Name: <%= item.name %> <br>
+      Item Quantity: <%= @invoice.find_invoice_item(@invoice, item).first.quantity %> <br>
```

Comment:
find_invoice_item.first.quantity should probably be a method that lives in the model with a name like find_quantity.

Reply:
I changed this to find the status and the quantity and added testing in the model.